### PR TITLE
Fixes Android not displaying non-scheduled notifications and Android notification size when title, subtitle or body not set

### PR DESCRIPTION
**Pull request by Danziger**
Fixes:
- Android notifications not showing up when options.at not set: https://github.com/EddyVerbruggen/nativescript-local-notifications/issues/109
- Android notifications always have the same size, even if no title, subtitle and body are set: https://github.com/EddyVerbruggen/nativescript-local-notifications/issues/110

### CODE REVIEW

**Comment by Danziger** [comment]
@EddyVerbruggen Do not merge this yet, I need to verify if I also need to change `NotificationRestoreReceiver#scheduleNotification`.

File: native-src/android/app/src/main/java/com/telerik/localnotifications/NotificationRestoreReceiver.java [hunk 1]
```diff
@@ -124,10 +124,19 @@ static void scheduleNotification(JSONObject options, Context context) throws Exc
...
+    // applyDeleteReceiver(options, context, builder);
     applyContentReceiver(options, context, builder);
 
     final Notification notification = builder.build();
+    final long triggerTime = options.getLong("atTime");
+
+    if (triggerTime == 0) {
+      // If we just want to show the notification immediately, there's no need to create an Intent,
+      // we just send the notification to the Notification Service:
+      ((NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE)).notify(options.optInt("id"), notification);
```
Comment (EddyVerbruggen) [inline native-src/android/app/src/main/java/com/telerik/localnotifications/NotificationRestoreReceiver.java]: Smart move 👏

File: src/local-notifications.ios.ts [hunk 2]
```diff
@@ -129,8 +129,9 @@ export class LocalNotificationsImpl extends LocalNotificationsCommon implements
       const {title, subtitle, body} = options;
       content.title = body || subtitle ? title : undefined;
       content.subtitle = body ? subtitle : undefined;
-      // On iOS, a notification with no body won't show up, so the subtitle or title will be used in this case.
-      content.body = body || subtitle || title;
+      // On iOS, a notification with no body won't show up, so the subtitle or title will be used in this case as body
+      // instead. If none of them is set, we set it to ' ' and will show up as an empty line in the notification:
+      content.body = body || subtitle || title || ' ';
```
Comment (EddyVerbruggen) [inline src/local-notifications.ios.ts]: Another smart move!

**Comment by EddyVerbruggen** [comment]
Excellent work, thanks a lot!

**Pull request merged**